### ISSUE TITLE
Possible misuse of aria-label

### ISSUE THREAD

**Issue by uxgary** [validator/validator#776]
To begin, I would like to say great work with the validate. My issue, however is that the validator flags "possible misuse of aria-label" when used on a `<section>` tag. For the sake of being ADA compliant, aria-labels are required on `<section>` tags as well as `<figure>` tags. This ensure the best user experience for people using screen readers. Siteimprove notes this as well as the following article: https://skryvets.com/blog/2017/12/24/make-your-site-accessible-step-by-step-instruction/#html5-section-does-have-aria-property

**Comment by Dan503** [comment]
Also an issue for the other sectioning elements `<nav>`, `<aside>`, and `<article>`.

**Comment by sideshowbarker** [comment]
I just now pushed a change that allows `aria-label` with `section` without warning.

That aligns with https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/

But the checker will still warn for `aria-label` on `figure` — because https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/ doesn’t mention anything about `figure`.

If you believe `aria-label` should be allowed for `figure`, please post a comment at https://github.com/w3c/html-aam/issues/160

> Also an issue for the other sectioning elements `<nav>`, `<aside>`, and `<article>`.

`nav` was already allowed without warning, and the change I just pushed now allows `aside` without warning. But it doesn’t allow `article` without warning.

If you believe `aria-label` should be allowed for `article`, please post a comment at https://github.com/w3c/html-aam/issues/160

**Comment by scottaohara** [comment]
@sideshowbarker, `article` should be allowed to have an `aria-label` and `aria-labelledby`.  That was already mentioned by @stevefaulkner in the HTML AAM issue you referenced, and is covered in [HTML AAM under the sectioning and grouping content](https://w3c.github.io/html-aam/#section-and-grouping-element-accessible-name-computation).

re: allowing `aria-label` on `figure` elements, there is an open todo to specify this in HTML AAM. However, I can tell you that due to [implementation gaps I outline here](https://www.scottohara.me/blog/2019/01/21/how-do-you-figure.html#wrapping-up), an `aria-label` is presently necessary to accurately convey a `figure`'s accessible name to the indicated screen reader and browser pairings.

**Comment by sideshowbarker** [comment]
> `article` should be allowed to have an `aria-label` and `aria-labelledby`. That was already mentioned by @stevefaulkner in the HTML AAM issue you referenced, and is covered in [HTML AAM under the sectioning and grouping content](https://w3c.github.io/html-aam/#section-and-grouping-element-accessible-name-computation)

OK, thanks — I’ve pushed a fix that allows it for `article` without warning

**Comment by sideshowbarker** [comment]
Closing per https://github.com/validator/validator/issues/776#issuecomment-491471730
